### ISSUE TITLE
Accordion: Accessibility issues that may require a breaking change

### ISSUE THREAD

**Issue by lyubomir-popov** [canonical-web-and-design/vanilla-framework#3318]
[The following are either difficult, require discussion, or are breaking changes, so separating from the [original list of validator issues](https://github.com/canonical-web-and-design/vanilla-framework/issues/3217)]

- [Rule:](https://www.w3.org/TR/wai-aria-practices/#wai-aria-roles-states-and-properties) Each accordion header button should be wrapped in an element with role heading (or h1/h2/h3/h4/h5/h6) that has a value set for aria-level that is appropriate for the information architecture of the page.
Each accordion header button is wrapped in an element with role heading that has a value set for aria-level that is appropriate for the information architecture of the page.

[Source](https://www.w3.org/TR/wai-aria-practices/#accordion)

Our accordion buttons are wrapped in `li`s, seems there should be a heasding in between:
![image](https://user-images.githubusercontent.com/2741678/91199982-7a837400-e6f6-11ea-8318-7d89904b0345.png)

We agreed not to tackle the above as part of the accessibility audit.
